OTVision/transform/transform.py

```diff
@@ -163,7 +163,7 @@ def read_tracks(tracks_file: Path) -> tuple[pd.DataFrame, dict]:
     """
 
     # Read dicts and turn tracks into DataFrame
-    tracks_dict = read_json(tracks_file, filetype=tracks_file.suffix)
+    tracks_dict = read_json(tracks_file, filetype=tracks_file.suffix, decompress=False)
```

Comment:
Wouldn't it be nice to automatically set `decompress` according to the input files? (file extension)

Reply:
See comment on next discussion.